packages/taquito/src/contract/estimate.ts

```diff
@@ -5,6 +5,44 @@ const MINIMAL_FEE_PER_GAS_MUTEZ = 0.1;
 
 const GAS_BUFFER = 100;
 
+/**
+ * Examples of use :
+ *
+ *  Estimate a transfer operation :
+ * ```
+ * // Assuming that provider and signer are already configured...
+ *
+ * const amount = 2;
+ * const address = 'tz1h3rQ8wBxFd8L9B3d7Jhaawu6Z568XU3xY';
+ *
+ * // Estimate gasLimit, storageLimit and fees for an transfer operation
+ * const est = await Tezos.estimate.transfer({ to: address, amount: amount })
+ * console.log(est.burnFeeMutez, est.gasLimit, est.minimalFeeMutez, est.storageLimit,
+ *  est.suggestedFeeMutez, est.totalCost, est.usingBaseFeeMutez)
+ *
+ * ```
+ *
+ * Estimate a contract origination :
+ * ```
+ * // Estimate fees related to originate the popular mutli-sig contract available here:
+ * // https://github.com/murbard/smart-contracts/blob/master/multisig/michelson/generic.tz
+ * // generic.json is referring to the JSON array extracted from the tezos-cli output
```

Comment:
Detailing the type of contract is irrelevant here. Our goal is to explain how to estimate an origination, not to distract the user with mutli-sig wallets. :)
Remove the extra details and focus on just estimation.